### DISCUSSION TITLE
"Convert to issues" on bullet point should carry sub-lists into new description

### DISCUSSION THREAD

**Discussion by knz** [community/community#4611] [Projects and Issues]
I used the "convert to issue" link on the first bullet point of [this issue](https://github.com/cockroachdb/cockroach/issues/67470)

![image](https://user-images.githubusercontent.com/642886/125300528-66eef500-e32a-11eb-95d2-49f556d38ad3.png)

However the description of [the new issue](https://github.com/cockroachdb/cockroach/issues/67482) was empty.

I had expected the description field of the new issue to contain a copy of the bullet points "under" the item in the original issue.

**Comment by gr2m** [comment]
I would also expect a description other then sub items to be carried. E.g. if I have this item

```md
- [ ] my task 

  Description here 
```

And I turn `my task` into a new issue, I'd expect the description to be removed from the tracking issue and moved into the description of the new "my task" issue


https://user-images.githubusercontent.com/39992/127549506-708fbd37-494e-4e52-aced-6ae07787aee2.mp4
